background.js

```diff
@@ -287,6 +300,34 @@ $(document).ready(function () {
       }
     });
   }
+  
+  /* method to change the anna status icon on page*/
+  function changeStatus(newStatus) {
+    
+    chrome.storage.local.get(/* String or Array */["statusicon"], function(items){
+      
+      console.log("items.statusicon == undefined: " +(items.statusicon == undefined));
+      console.log("items.statusIcon == \"false\": "+(items.statusIcon == "false") )
+      if((items.statusicon == undefined) ||( items.statusicon == "false") ){
+        newStatus = "noIcon";
+      }
+
+      chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) { 
+
+        var tabid = tabs[0].id;
+        
+          chrome.tabs.executeScript(tabid, {
```

Comment:
This part is giving me a lot of console errors when i fetched the pull request, can you check it out .
` Error in response to tabs.query: TypeError: Cannot read property 'id' of undefined
    at Object.callback (chrome-extension://kefoeekdpbolcmhgplhgmgofcopopaai/background.js:317:29)
    at Object.callback (chrome-extension://kefoeekdpbolcmhgplhgmgofcopopaai/background.js:315:19) `

Reply:
This is error because, here, the extension is trying to inject script in the page "background page" (not actual webpage), that's why it throwing errors(because it cannot read tabId of that page). 
It works fine when background page is "Not Open", and that is exactly what we want.